docs/src/main/sphinx/connector/hive.md

```diff
@@ -302,6 +302,9 @@ Hive connector documentation.
 * - `hive.max-partition-drops-per-query`
   - Maximum number of partitions to drop in a single query.
   - 100,000
+* - `hive.metastore.partition-batch-size.max`
+  - Maximum number of partitions processed in a single batch.
```

Comment:
I guess users don't understand what "batch" means from this description?

Reply:
Hmm.. maybe `Maximum number of partitions that the Hive Metastore processes together at one time (in a group, or ‘batch’).` ?

Reply:
I think batch is fine .. could also say in a single group of operations or so .. but you are really just defining the word batch. Lets merge this as it stands.